### PR TITLE
Ccn 42 cartview zd

### PR DESCRIPTION
**Pull request by Labedlam**
there are some changes to product details in this branch

### CODE REVIEW

File: productDetail/quantityInput/templates/quantityInput.tpl.html [hunk 1]
```diff
@@ -1,23 +1,54 @@
...
+            <select id="itemQuantity"
+                    name="itemQuantity"
+                    ng-required="true"
+                    ng-if="item.StandardPriceSchedule.RestrictedQuantity"
+                    class="form-control text-center"
+                    ng-model="item.Quantity"
+                    ng-options="priceBreak.Quantity as priceBreak.Quantity for priceBreak in item.StandardPriceSchedule.PriceBreaks"></select>
+            <div class="form-control-feedback" ng-show="item.QuantityMultiplier > 1">{{'x' + item.QuantityMultiplier}}</div>
+    </div>
+    <div>
```
Comment (robertsoniv) [inline productDetail/quantityInput/templates/quantityInput.tpl.html]: Get rid of this extra <div>

File: productDetail/productDetail.js [hunk 2]
```diff
@@ -17,28 +17,31 @@ function ProductConfig($stateProvider) {
...
+               $exceptionHandler(error);
+            })
     };
 
     vm.findPrice = function(qty){
-        var finalPriceBreak = null;
         angular.forEach(vm.item.StandardPriceSchedule.PriceBreaks, function(priceBreak) {
             if (priceBreak.Quantity <= qty)
-                finalPriceBreak = angular.copy(priceBreak);
+                vm.finalPriceBreak = angular.copy(priceBreak);
```
Comment (robertsoniv) [inline productDetail/productDetail.js]: What was the reason behind this change?

File: cart/templates/cart.tpl.html [hunk 3]
```diff
@@ -68,5 +69,7 @@ <h5 ng-repeat="promotion in cart.promotions.Items">
             <ordercloud-promotion-input order="cart.order" orderpromotions="cart.promotions"></ordercloud-promotion-input>
```
Comment (robertsoniv) [inline cart/templates/cart.tpl.html]: Does this need to be here anymore?

File: productDetail/productDetail.js (hunk 2 above)
Comment (Labedlam) [inline productDetail/productDetail.js]: when I was writing a unit test for it. It was easier to get access to the var finalPriceBreak when it was tied to the scope.

File: cart/templates/cart.tpl.html (hunk 3 above)
Comment (Labedlam) [inline cart/templates/cart.tpl.html]: No, I'll remove it.

**Pull request merged**